packages/basti-cdk/src/basti-access-security-group.ts

```diff
@@ -1,63 +1,67 @@
-import { aws_ec2 } from 'aws-cdk-lib';
+import { aws_ec2, Tags } from 'aws-cdk-lib';
 
 import { generateShortId } from './basti-helper';
-import { TARGET_ACCESS_SECURITY_GROUP_NAME_PREFIX } from './basti-constants';
+import {
+  BASTION_INSTANCE_CREATED_BY_TAG_NAME,
+  TARGET_ACCESS_SECURITY_GROUP_NAME_PREFIX,
+} from './basti-constants';
 
 import type { Construct } from 'constructs';
 import type { IBastiInstance } from './basti-instance';
 
 /**
- * The properties for the bastion access security group.
+ * The properties for the Basti access security group.
```

Comment:
I prefer to use the name _Basti_ when referring to the whole configured Basti instance. I'd use the term _bastion instance_ when referring to the particular EC2 instance inside the Basti instance.